cluster.yml

```diff
@@ -6,13 +6,21 @@
   import_playbook: legacy_groups.yml
 
 - hosts: bastion[0]
+  pre_tasks:
+  - include_vars:
+      dir: ../../vars/kubespray
+    tags: always
```

Comment:
We shouldn't have this here. this repo doesn't have a `vars` folder, the `ansible-app` repo has it.